app/factory.py

```diff
@@ -29,7 +31,7 @@ def create_app(app_name=a_name, **kwargs):
     # register the CLI admin blueprint
     app.register_blueprint(admin)
     # Allow CORS
-    CORS(app, origins='*', expose_headers=['Content-Disposition'])
+    CORS(app, origins='*', expose_headers=['Content-Disposition'], send_wildcard=True)
```

Comment:
FYI: this contains the change to CORS that was believed to be a potential issue but not deployed